trailblazer/server/api.py

```diff
@@ -123,7 +123,7 @@ def update_analysis(analysis_id):
     """Update a specific analysis"""
     try:
         process = multiprocessing.Process(
-            target=store.update_run_status, kwargs={"analysis_id": analysis_id, "ssh": True}
+            target=store.update_run_status, kwargs={"analysis_id": analysis_id}
```

Comment:
Out of curiosity, what happens here? From the API we interact with the database via multiprocessing?

Reply:
Seems like

Reply:
Actually slurm on Hasta